packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -351,12 +351,21 @@ abstract class ElementPatchOperation<V> implements ElementOperation {
   abstract toJSON(): Dict<string>;
 }
 
+export const STYLE_WARNING = `Binding style attributes may introduce cross-site scripting vulnerabilities; please ensure that values being bound are properly escaped.`
+
+function styleWarning(name) {
+  if (name === 'style') {
+    console.warn(STYLE_WARNING);
```

Comment:
We will need to be able to provide the environment with the mechanism for warning I think...